src/DistributedLock.Core/Internal/DistributedLockHelpers.cs

```diff
@@ -133,6 +134,30 @@ public static bool TryUpgradeToWriteLock(IDistributedLockUpgradeableHandle handl
         SyncViaAsync.Run(t => t.handle.TryUpgradeToWriteLockAsync(t.timeout, t.cancellationToken), (handle, timeout, cancellationToken));
     #endregion
 
+    #region ---- IDbSynchronizationStrategy implementations ----
+    public static ValueTask<THandle?> TryAcquireAsync<THandle>(IDbSynchronizationStrategy<THandle> strategy, DatabaseConnection connection, string resourceName, TimeSpan timeout, CancellationToken cancellationToken)
+        where THandle : class =>
+        strategy.TryAcquireAsync(connection, resourceName, timeout, cancellationToken);
+
+    public static ValueTask<THandle> AcquireAsync<THandle>(IDbSynchronizationStrategy<THandle> strategy, DatabaseConnection connection, string resourceName, TimeSpan? timeout, CancellationToken cancellationToken)
+        where THandle : class =>
+        strategy.TryAcquireAsync(connection, resourceName, timeout, cancellationToken).ThrowTimeoutIfNull();
+
+    public static THandle Acquire<THandle>(IDbSynchronizationStrategy<THandle> strategy, DatabaseConnection connection, string resourceName, TimeSpan? timeout, CancellationToken cancellationToken)
+        where THandle : class =>
+        SyncViaAsync.Run(
+            state => AcquireAsync(state.strategy, state.connection, state.resourceName, state.timeout, state.cancellationToken),
+            (strategy, connection, resourceName, timeout, cancellationToken)
+        );
+
+    public static THandle? TryAcquire<THandle>(IDbSynchronizationStrategy<THandle> strategy, DatabaseConnection connection, string resourceName, TimeSpan timeout, CancellationToken cancellationToken)
+        where THandle : class =>
+        SyncViaAsync.Run(
+            state => TryAcquireAsync(strategy, connection, resourceName, timeout, cancellationToken),
+            (strategy, connection, resourceName, timeout, cancellationToken)
+        );
+    #endregion
```

Comment:
Let's remove these helper methods. They're just calling `strategy.TryAcquire(Async)` which can be called directly by `PostgresDistributedLock`

Reply:
Removed